File: Converge/Models/BuildingSearchInfo.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using Microsoft.Graph;
using System.Collections.Generic;

namespace Converge.Models
{
    public class BuildingSearchInfo
    {
        public List<BuildingBasicInfo> BuildingInfoList { get; set; }

        public BuildingSearchInfo(List<BuildingBasicInfo> buildingInfoList)
        {
            BuildingInfoList = buildingInfoList;
        }
    }
}

File: Converge/ClientApp/src/types/BuildingSearchInfo.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
import BuildingBasicInfo from "./BuildingBasicInfo";
import QueryOption from "./QueryOption";

interface BuildingSearchInfo {
  buildingInfoList: BuildingBasicInfo[];
  skipToken:QueryOption;
}

export default BuildingSearchInfo;
